File: reopengles/src/main/java/com/ray/reopengles/MainActivity.java
package com.ray.reopengles;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;

import com.ray.reopengles.simplegraph.GraphOptionActivity;
import com.ray.reopengles.simplegraph.SimpleGraphActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void clickSimpleGraph(View view) {
        GraphOptionActivity.launch(this);
    }

    public void clickTexture(View view) {
        SimpleGraphActivity.launch(this, 2);
    }

}

File: view/src/main/java/com/hikobe8/view/MainActivity.kt
package com.hikobe8.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*
import kotlin.concurrent.thread

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun dashboard(view: View) {
        DashboardActivity.launch(this)
    }

    fun piechart(view: View) {
        PieChartActivity.launch(this)
    }

}
